### ISSUE TITLE
upgrade to 0.13 

### ISSUE THREAD

**Issue by ethereumdegen** [Neopallium/bevy_water#27]

**Comment by Neopallium** [comment]
Updated to support bevy 0.13, but it will need to wait for the 0.13.1 fix release.  The water shader depends on globals for the time, which was fixed here: https://github.com/bevyengine/bevy/pull/12032

**Comment by Neopallium** [comment]
I went ahead and published version 0.13.0 to support bevy v0.13.1

You will need:
```
[patch.crates-io]
bevy = { git = "https://github.com/bevyengine/bevy.git", branch = "release-0.13.1" }
```
In your `Cargo.toml` file until bevy v0.13.1 is released.
